Update the book and examples to Twilight 0.11 (#38)
dawn-rs/dawn-rs.github.io: src/chapter_1_crates/section_7_first_party/section_4_util.md

### Builder

> The `builder` feature enables builders for large structs. There are as
> of writing builders for [`Command`], [`Embed`] and
> [`InteractionResponseData`].

Reviewer: I think it's better to link to the builders themselves:
```suggestion
The `builder` feature enables builders for large structs. At the time of
writing, it contains the following builders:
- [`CommandBuilder`]
- [`EmbedBuilder`]
- [`InteractionResponseData`]
```

*docs*: <https://docs.rs/twilight-util>

*crates.io*: <https://crates.io/crates/twilight-util>

> 
> [`Command`]: https://api.twilight.rs/twilight_model/application/command/struct.Command.html
> [`Embed`]: https://api.twilight.rs/twilight_model/channel/embed/struct.Embed.html
> [`InteractionResponseData`]: https://api.twilight.rs/twilight_model/http/interaction/struct.InteractionResponseData.html

Reviewer: ```suggestion
[`CommandBuilder`]: https://api.twilight.rs/twilight_util/builder/command/struct.CommandBuilder.html
[`EmbedBuilder`]: https://api.twilight.rs/twilight_util/builder/embed/struct.EmbedBuilder.html
[`InteractionResponseDataBuilder`]: https://api.twilight.rs/twilight_util/builder/struct.InteractionResponseDataBuilder.html
```
